### DISCUSSION TITLE
Fetching build files from AWS s3

### DISCUSSION THREAD

**Discussion by Biplo12** [jeffreylanters/react-unity-webgl#497] [Question and Answer]
Hi, i have a question, is it posible to fetch build files from s3 domain and pass it to the useUnityContext?

**Comment by Bunderant** [comment]
Yep, that's the way we've been using it where I work. We just change the `bucket` variable depending where we're hosting, either local, staging, or production. Just make sure the metadata is set up correctly in S3 for your uploaded files, mainly Content-Type and Content-Encoding.

We use a CloudFront URL for our production bucket. Still trying to think of a way to have some pattern matching for these URLs, would be nice not to have to redeploy the server when the filenames change (if you're appending hashes, for instance, which is useful to get around CloudFront caching when you release an update). 

EDIT: Added streamingAssetsUrl

```
const bucket = 'https://some.s3.url/Bucket';

const unityContext = new UnityContext({
	loaderUrl: `${bucket}/my-project.loader.js`,
	dataUrl: `${bucket}/my-project.data.gz`,
	frameworkUrl: `${bucket}/my-project.framework.js.gz`,
	codeUrl: `${bucket}/my-project.wasm.gz`,
	streamingAssetsUrl: `${bucket}/StreamingAssets`
});
```

**Reply by Magvin** [reply]
I think you also need include streamingAssetsUrl

**Reply by Bunderant** [reply]
Yeah, generally you would, in our case the web player is pretty light and we don't have anything stored in StreamingAssets. 

Thanks for mentioning that, though, probably good to note for example's sake. I updated the original code sample to include that.
